packages/taquito/test/batch/rpc-batch-provider.spec.ts

```diff
@@ -566,6 +566,45 @@ describe('OperationBatch test', () => {
     });
   });
 
+  describe('withIncreasePaidStorage batch operation', () => {
+    it('should produce an operation batch which contains an increasePaidStorage operation', async (done) => {
```

Comment:
could you add the following test case as well:
- should produce a batch operation which contains an increasePaidStorage operation where fee, gas limit and storage limit are specified by the user
- should produce a batch operation which contains a reveal and an increasePaidStorage operation